### DISCUSSION TITLE
Paper reference for Diff-in-diff DML Score function

### DISCUSSION THREAD

**Discussion by RShi8** [DoubleML/doubleml-for-py#253] [General] [answered]
Hello, I am new to this topic. Could anyone provide a paper reference for the Neyman orthogonal score function for the diff-in-diff repeated cross-sectional data? The score function in question is listed in the User Guide Section 5.2.6. https://docs.doubleml.org/stable/guide/scores.html. I know one of the reference paper is Chang (2020), but the score function in this paper seems to differ from the score function written in the User Guide. Thus, I would like to know more about the theoretical background for the score function being used for this DML package. Thank you.

**Comment by SvenKlaassen** [comment, marked as answer]
Hi @RShi8,

the corresponding reference is [Zimmer et al. (2018)](https://arxiv.org/abs/1809.01643). 
The documentation is not optimal (maybe i can find time to update this in the future), but the references are listed in the [Example Notebook](https://docs.doubleml.org/stable/examples/py_double_ml_did.html).
The score is identical to the score of [Sant’Anna and Zhao (2020)](https://doi.org/10.1016/j.jeconom.2020.06.003), when using linear models.
